PowerShellBuild/Public/Initialize-PSBuild.ps1

```diff
@@ -13,10 +13,17 @@ function Initialize-PSBuild {
     #>
     [cmdletbinding()]
     param(
+        [Parameter(Mandatory)]
+        [Hashtable]
+        $BuildEnvironment,
```

Comment:
@pauby. Can you add some comment-based help to this? Missing help for parameters causes the tests to fail.

Reply:
Done.